### PR TITLE
MAR-113:

### PR DESCRIPTION
**Pull request by fericsepi**
- new partial called callout which outputs a panel on a template,
 - splitting index.scss to allow loading files from the /static/ folder locally
 - WIP: view and template for add barrier
 - WIP: barrier status view

### CODE REVIEW

File: templates/partials/callout.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {% endif %}
+    {% if callout.text %}
+        <p class="callout__text">{{ callout.text }}</p>
+    {% endif %}
+
+    {#  Button  #}
+    {% if callout.button %}
+        {#    FORM    #}
+        {% if callout.button.action %}
+            <form action="{{ callout.button.action }}" method="{{ callout.button.method }}">
```
Comment (dankingston) [inline templates/partials/callout.html]: In almost all of the forms I've looked at so far, the form posts to the same page, so we can leave the action blank. But it depends how customisable we want this component to be.

File: templates/partials/callout.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                {% if callout.button.csrf_token %}
+                    <input type="hidden" name="_csrf" value="{{ callout.button.csrfToken }}">
```
Comment (dankingston) [inline templates/partials/callout.html]: We should be able to forget the button's `csrf_token` parameter and just use `{% csrf_token %}` here.

Comment (fericsepi) [reply templates/partials/callout.html]: just matched whatever logic they implemented in node

**Comment by fericsepi** [comment]
Thanks Dan.
Yeah, they have a few callouts in there, hence the component.
I noticed that there are a few components that are only used once with loads of logic in the view so there's definitely room to simplify things.

File: templates/partials/callout.html (hunk 2 above)
Comment (fericsepi) [reply templates/partials/callout.html]: good catch :)

**Pull request merged**